mlir/test/Integration/Dialect/Vector/CPU/ArmSME/test-outerproduct-f32.mlir

```diff
@@ -14,6 +14,12 @@
 // REDEFINE: %{entry_point} = test_outerproduct_with_accumulator_4x4xf32
 // RUN: %{compile} | %{run} | FileCheck %s --check-prefix=WITH-ACC
```

Comment:
[nit] It would be good to only compile once, see e.g. https://github.com/llvm/llvm-project/blob/640274ff1fe17852615564a80d9f8c69b67f1d59/mlir/test/Integration/Dialect/Vector/CPU/ArmSVE/test-contraction.mlir#L1-L20